ami/AMI.md

```diff
@@ -11,5 +11,5 @@ Here is an example on how to install Mesos 1.5.0 using maws (https://github.com/
 
 ```bash
 $(maws login "Team 10")
-AWS_PROFILE=273854932432_Mesosphere-PowerUser packer build -color -var 'ami_name=JenkinsMarathonCI-Debian8-2018-02-02' -var 'mesos_version=1.5.0-2.0.1' marathon-jenkins-ami.json
+AWS_PROFILE=273854932432_Mesosphere-PowerUser packer build -color -var 'ami_name=JenkinsMarathonCI-Debian9-2018-02-02' -var 'mesos_version=1.5.0-2.0.1' marathon-jenkins-ami.json
```

Comment:
why do we have Debian9-2018-02-09 in jenkinsfile and Debian9-2018-02-02 here?

Reply:
This is just an example.